fe/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -530,6 +530,146 @@ public void init() {
                             "_ZN5doris15BitmapFunctions17bitmap_update_intIN9doris_udf6IntValEEEvPNS2_15FunctionContextERKT_PNS2_9StringValE")
                     .build();
 
+    private static final Map<Type, String> BITMAP_INTERSECT_INIT_SYMBOL =
```

Comment:
This way of adding functions is too ugly. We should improve it in the future.

Reply:
OK. I agree with you.  I think the function manage way should also be improved.